runtime/syntax/qml.vim

```diff
@@ -0,0 +1,1145 @@
+" Vim syntax file
+" Language:     QML
+" Maintainer:   Peter Hoeg <peter@hoeg.com>
```

Comment:
This should be changed to "Previous Maintainer:" assuming Peter won't be maintaining the Vim distributed file.